corehq/apps/hqadmin/reports.py

```diff
@@ -333,3 +337,146 @@ def _shorten_and_hyperlink_commit(self, commit_sha):
                 abbrev_sha=commit_sha[:7]
             )
         return None
+
+
+class UCRRebuildRestrictionTable:
+    UCR_RESTRICTION_THRESHOLD = 1_000_000
+
+    selected_domain: str
+    restriction_ff_status: str
+
+    def __init__(self, *args, **kwargs):
+        self.selected_domain = kwargs.get('selected_domain')
```

Comment:
you can select only one domain?

Reply:
As of now, yes.

Reply:
I had a chat with AE and the domain filter is redundant.

Reply:
@Charl1996 Could you please update the screenshot in the PR description to show the updated report filters since the domain filter has now been removed?

Reply:
Yes! Let me do that now

Reply:
Thanks!